xline-client/src/clients/cluster.rs

```diff
@@ -0,0 +1,17 @@
+// TODO: Remove these when the placeholder is implemented.
+#![allow(missing_copy_implementations)]
+#![allow(clippy::new_without_default)]
```

Comment:
this PR does not need these lines, please add them when you need them, same as other files

Reply:
The files act as placeholders. I need these to satisfy the crate clippy lints.